apps/console/src/features/organizations/constants/organization-constants.ts

```diff
@@ -49,3 +54,25 @@ export enum ORGANIZATION_TYPE {
     STRUCTURAL = "STRUCTURAL",
     TENANT = "TENANT"
 }
+
+
+// Role constants
```

Comment:
Check formatting

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/81488c55dd6c9c7a63575dee78e71f2b80643963